scripts/algorithms/StepGLM.dml

```diff
@@ -173,15 +173,15 @@ if (dir == "forward") {
 		# Subsequent passes over the features
 		parfor (i in 1:num_features) { 
 			if (as.scalar(columns_fixed[1,i]) == 0) {	
-        
+		
 				# Construct the feature matrix
 				X = cbind (X_global, X_orig[,i]);
-        
```

Comment:
Are tabs our standard for sure? I see a lot of DML files with spaces.

Reply:
The reason why i changed this in this file was because it was inconsistent, and i was debugging why it was crashing.
I should probably revert this, because now that i look at it, it is not done correctly either by me.